### PR TITLE
Feature/sample kainos projects

### PR DESCRIPTION
**Pull request by adrianz-kainos**

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -125,3 +125,10 @@ $(document).ready(function() {
   toggleContent.showHideCheckboxToggledContent();
 
 });
+
+// Used in tables to collapse child rows
+// TODO: make it expand a single row (under the clicked one) instead of triggering all of them in a single click
```
Comment (xjedam) [inline app/assets/javascripts/application.js]: Please remove that and record as an issue so we don't loose track of that

File: app/assets/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+/javascripts/
```
Comment (xjedam) [inline app/assets/.gitignore]: are you sure ignoring all javascript assets is a good idea?

File: app/views/project.html [hunk 3]
```diff
@@ -58,15 +61,98 @@ <h3 class="heading-small">Components</h3>
...
 		</section>
 	</div>
-</main>
+	
+	<div class="grid-row">
+		<section class="column-half">
+			<h2 class="heading-medium">Kainos team</h2>
+			<div>
+			{% if data["ourTeam"] %}
+			{# TODO: put the tables in a template #}
```
Comment (xjedam) [inline app/views/project.html]: TODO :(

File: app/views/project.html [hunk 4]
```diff
@@ -58,15 +61,98 @@ <h3 class="heading-small">Components</h3>
...
+			   <td>{{ teamMember.role}}</td>
+			   <td>{{ teamMember.name }}</td>
+			   <td data-collapsable-parent="1">...</td>
+			 </tr>
+			 <tr>
+			   <td data-collapsable-child="1" colspan="3">
+			     <div class="grid-row">
+			       <section class="column-half">
+			         <p>Primary location: {{ teamMember.primaryLocation }} </p>
+			         <p>Secondary location: {{ teamMember.secondaryLocation}} </p>
```
Comment (xjedam) [inline app/views/project.html]: consider what happens if none secondary location is specified

File: app/views/project.html [hunk 5]
```diff
@@ -58,15 +61,98 @@ <h3 class="heading-small">Components</h3>
...
+			       </section>
+			       <section class="column-half">
+			         <p>Email: {{ teamMember.email}} </p>
+			         <p>Mobile: {{ teamMember.mobile}} </p>
```
Comment (xjedam) [inline app/views/project.html]: consider what happens if no mobile specified

File: app/views/project.html [hunk 6]
```diff
@@ -58,15 +61,98 @@ <h3 class="heading-small">Components</h3>
...
+			   <td>{{ teamMember.role}}</td>
+			   <td>{{ teamMember.name }}</td>
+			   <td data-collapsable-parent="1">...</td>
+			 </tr>
+			 <tr>
+			   <td data-collapsable-child="1" colspan="3">
+			     <div class="grid-row">
+			       <section class="column-half">
+			         <p>Primary location: {{ teamMember.primaryLocation }} </p>
+			         <p>Secondary location: {{ teamMember.secondaryLocation}} </p>
```
Comment (xjedam) [inline app/views/project.html]: same logic applies here - we may not have this kind of data

File: app/views/project.html [hunk 7]
```diff
@@ -58,15 +61,98 @@ <h3 class="heading-small">Components</h3>
...
+			       </section>
+			       <section class="column-half">
+			         <p>Email: {{ teamMember.email}} </p>
+			         <p>Mobile: {{ teamMember.mobile}} </p>
```
Comment (xjedam) [inline app/views/project.html]: and here

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (adrianz-kainos) [inline app/assets/javascripts/application.js]: Done.

File: app/assets/.gitignore (hunk 2 above)
Comment (adrianz-kainos) [inline app/assets/.gitignore]: Sorry, Eclipse generated it and I overlooked it. Removed, thx.

File: app/views/project.html (hunk 3 above)
Comment (adrianz-kainos) [inline app/views/project.html]: Done.

File: app/views/project.html (hunk 5 above)
Comment (adrianz-kainos) [inline app/views/project.html]: If statements added.

File: app/views/project.html (hunk 4 above)
Comment (adrianz-kainos) [inline app/views/project.html]: If statements added.

File: app/views/project.html (hunk 6 above)
Comment (adrianz-kainos) [inline app/views/project.html]: If statements added.

File: app/views/project.html (hunk 7 above)
Comment (adrianz-kainos) [inline app/views/project.html]: If statements added.

**Pull request merged**